torch_geometric/nn/conv/sage_conv.py

```diff
@@ -83,8 +84,28 @@ def __init__(
             in_channels = (in_channels, in_channels)
 
         if aggr == 'lstm':
-            kwargs['aggr_kwargs'] = dict(in_channels=in_channels[0],
-                                         out_channels=in_channels[0])
+            kwargs.setdefault('aggr_kwargs', {})
+            kwargs['aggr_kwargs'].setdefault('in_channels', in_channels[0])
+            kwargs['aggr_kwargs'].setdefault('out_channels', in_channels[0])
+
+        aggr_out_channels = in_channels[0]
+        if isinstance(aggr, List) and len(aggr) > 1:
+            kwargs.setdefault('aggr_kwargs', {})
+            kwargs['aggr_kwargs'].setdefault('mode', 'cat')
+            kwargs['aggr_kwargs'].setdefault('mode_kwargs', {})
+            combine_mode = kwargs['aggr_kwargs']['mode']
+            if combine_mode == 'cat':
+                aggr_out_channels = in_channels[0] * len(aggr)
+            if combine_mode == 'proj' or combine_mode == 'attn':
```

Comment:
Here again. Let‘s try to not add any custom combine mode logic here. If the user sets a custom combine mode. They will be responsible for setting mode_kwargs as well.